stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift

```diff
@@ -0,0 +1,109 @@
+#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+import Darwin
+#else
+import Glibc
+#endif
+import CTensorFlow
+import CTensorFlowUnittest
+import TensorFlow
+import StdlibUnittest
+
+typealias CTFServer = OpaquePointer
+
+@usableFromInline
```

Comment:
Why do you need `@usableFromInline`? What this attribute does is that it makes a function have public linkage in binary. This is only necessary when this is to be called from an `@inlinable` context, which I don't think your code is doing.

Reply:
I moved this into the class and removed the @inlinable attribute.